src/Aspire.Dashboard/Otlp/Model/OtlpSpanEvent.cs

```diff
@@ -13,6 +13,7 @@ public class OtlpSpanAttributeItem(string name, string value) : IPropertyGridIte
 
 public class OtlpSpanEvent(OtlpSpan span) : IPropertyGridItem
 {
+    public required Guid InternalId { get; init; }
```

Comment:
GUID is 128 bit, and I assume we can have a lot of these in the system. Did you consider using an incrementing `UInt32`? It'd take less space, and be faster to compute.